src/bepasty/templates/display.html

```diff
@@ -76,4 +101,12 @@ <h1>
 <script src="{{ url_for('bepasty.xstatic', name='bootbox', filename='bootbox.min.js') }}" type="text/javascript"></script>
 <script src="{{ url_for('bepasty.xstatic', name='asciinema_player', filename='asciinema-player.js') }}" type="text/javascript"></script>
 <script src="{{ url_for('static', filename='app/js/utils.js') }}" type="text/javascript"></script>
+{% if may(EDIT) %}
+<script>
+    <!-- function that used by utils.js -->
+    function contenttype_autocomplete(modal_box) {
+        {{ utils.contenttype_autocomplete('$(modal_box).find("#contenttype")', contenttypes) }}
+    };
+</script>
+{% endif %}
```

Comment:
could this function definition just be in utils.js and not inlined here?

guess its definition does not need to depend on "may edit"?

Reply:
this extends contenttypes list from python. so maybe more complex to pass it to javascript

Reply:
a right, it's calling py code via templating... - keep it as is.